source/jormungandr/jormungandr/autocomplete/geocodejson.py

```diff
@@ -277,12 +277,21 @@ def make_url(self, end_point, uri=None):
         return url
 
     def basic_params(self, instances):
+        '''
```

Comment:
Have you installed pre-commit? Not sure black likes it this way...

Reply:
black is run by travis, and it seems to like it. 
In navitia black is run with: `--skip-string-normalization` to prevent every single quoted string (`''`) to be changed in a double quoted string (`""`) as the former was the recommended way before we introduced black.